packages/react-topology/src/utils/useCollapsibleGroup.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import { observer } from 'mobx-react';
+import { Node } from '../types';
+import { ShapeProps } from './useCustomNodeShape';
+
+export interface WithCollapsibleGroupProps {
+  collapsible: boolean;
+  collapsedWidth?: number;
+  collapsedHeight?: number;
+  onCollapseChange?: (group: Node, collapsed: boolean) => void;
+  getCollapsedShape?: (node: Node) => React.FC<ShapeProps>;
+  collapsedShadowOffset?: number; // defaults to 10
+}
+
+export const useCollapsibleGroup = (
```

Comment:
What is the purpose this hook? Seems useless to just return the values given to it.